subsystem/cluster_test.go

```diff
@@ -2372,6 +2428,39 @@ func registerHostsAndSetRoles(clusterID strfmt.UUID, numHosts int) []*models.Hos
 		})
 		Expect(err).ShouldNot(HaveOccurred())
 	}
+	generateConnectivityPostStepReply := func(h *models.Host, connectivityReport *models.ConnectivityReport) {
```

Comment:
Why copy this function and not write it once?

Reply:
Will change